### PR TITLE
feat(StatusSeedPhraseInput): Adding component documentation

### PR DESCRIPTION
**Pull request by alexandraB99**
Also fixed component's height in demonstration page

Part of #616

### Checklist

- [x] follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
  - the scope should be the component's name e.g: `feat(StatusListItem): ... `
  - when adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [x] add documentation if necessary (new component, new feature)
- [x] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?
- [ ] is this a breaking change?
    - [ ] use the dedicated `BREAKING CHANGE` commit message section
    - [ ] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop)
        - [ ] (pre-merge) adapt code to breaking changes
        - [ ] (post-merge) update StatusQ submodule pointer
- [ ] test changes in [status-desktop](https://github.com/status-im/status-desktop)

### CODE REVIEW

File: src/StatusQ/Controls/StatusSeedPhraseInput.qml [hunk 1]
```diff
@@ -40,24 +40,58 @@ import StatusQ.Core.Theme 0.1
...
+        This property is an alias to the StatusInput's textEdit property.
+    */
     property alias textEdit: seedWordInput
+    /*!
+        \qmlproperty text
+        This property is an alias to the StatusInput's text property.
+    */
     property alias text: seedWordInput.text
+    /*!
+        \qmlproperty text
```
Comment (noeliaSD) [inline src/StatusQ/Controls/StatusSeedPhraseInput.qml:63]: Should this be `leftComponentText`?

File: src/StatusQ/Controls/StatusSeedPhraseInput.qml [hunk 2]
```diff
@@ -40,24 +40,58 @@ import StatusQ.Core.Theme 0.1
...
+        \qmlproperty filteredList
+        This signal is emitted when the user selects a word from
+        the suggestions list, either by clicking on it or by completing
+        typing 4 charactersnd passes as a parameter the selected word.
+        The corresponding handler is \c onDoneInsertingWord
+    */
     signal doneInsertingWord(string word)
-    /*
-        This signal is emitted when the user pressed a keyboard key and passes as a
+    /*!
```
Comment (noeliaSD) [inline src/StatusQ/Controls/StatusSeedPhraseInput.qml:86]: If you use a specific tag, like for properties but for signals, `\qmlsignal`, that classifies them in a certain way in the docu file.

File: src/StatusQ/Controls/StatusSeedPhraseInput.qml [hunk 3]
```diff
@@ -40,24 +40,58 @@ import StatusQ.Core.Theme 0.1
...
+
    For a list of components available see StatusQ.
 */
 
 Item {
     id: root
     width: 162
     height: 44
+    /*!
+        \qmlproperty textEdit
```
Comment (noeliaSD) [inline src/StatusQ/Controls/StatusSeedPhraseInput.qml:53]: General for all properties: If you use the syntax `StatusSeedPhraseInput::nameOfProperty` the docu makes that text clickable and navigable. Same if you have a method or signal.
It is also common to add the property type: `\qmlproperty propertyType Type::nameOfProperty`

Comment (alexandraB99) [reply src/StatusQ/Controls/StatusSeedPhraseInput.qml]: done

File: src/StatusQ/Controls/StatusSeedPhraseInput.qml (hunk 2 above)
Comment (alexandraB99) [reply src/StatusQ/Controls/StatusSeedPhraseInput.qml:88]: done

File: src/StatusQ/Controls/StatusSeedPhraseInput.qml (hunk 1 above)
Comment (alexandraB99) [reply src/StatusQ/Controls/StatusSeedPhraseInput.qml]: done

**Review by noeliaSD — APPROVED** [review]

**Review by iurimatias — APPROVED** [review]

**Pull request merged**